vagrant_box_defaults.rb

```diff
@@ -3,12 +3,12 @@
 Vagrant.require_version ">= 2.2.0"
 
 $SERVER_BOX = "cilium/ubuntu-dev"
-$SERVER_VERSION= "208"
+$SERVER_VERSION= "213"
 $NETNEXT_SERVER_BOX= "cilium/ubuntu-next"
-$NETNEXT_SERVER_VERSION= "105"
+$NETNEXT_SERVER_VERSION= "114"
 @v54_SERVER_BOX= "cilium/ubuntu-5-4"
-@v54_SERVER_VERSION= "12"
+@v54_SERVER_VERSION= "19"
 @v419_SERVER_BOX= "cilium/ubuntu-4-19"
-@v419_SERVER_VERSION= "48"
+@v419_SERVER_VERSION= "53"
 @v49_SERVER_BOX= "cilium/ubuntu"
-@v49_SERVER_VERSION= "208"
+@v49_SERVER_VERSION= "213"
```

Comment:
These updates need to be in a PR with a specific name. A PR is already opened at https://github.com/cilium/cilium/pull/17761 for that, but a couple CI tests are consistently failing. I'm happy to update the box versions in that PR once I've debugged CI.

Reply:
Did not know, dropped the last commit :-)